gnc/navigator_controller/nodes/mrac_controller.py

```diff
@@ -96,11 +96,15 @@ def __init__(self):
         self.lin_vel = np.zeros(2)
         self.ang_vel = 0
         self.state = Odometry()
+
+        self.use_lqrrt = True
+
         # Subscribers
-        if self.use_external_tgen:
-            rospy.Subscriber("/trajectory", PoseTwistStamped, self.set_traj)
+        if self.use_lqrrt:
+            rospy.Subscriber("/lqrrt/ref", Odometry, self.set_traj_from_odom_msg)
         else:
-            rospy.Subscriber("/waypoint", PoseStamped, self.set_waypoint)
+            rospy.Subscriber("/trajectory", PoseTwistStamped, self.set_traj)
+
         rospy.Subscriber("/odom", Odometry, self.get_command)
         rospy.Subscriber('/learn', Bool, self.toggle_learning)
```

Comment:
MRAC is nearly 1 year old and the first thing I ever attempted to ROSify.
Will be restructuring / cleaning it quite a bit (and the GNC folder in general, excluding perception) over the next two weeks. Using services / actions more will be part of that reform.